exe/shell/main.go

```diff
@@ -359,6 +369,28 @@ func addCmd(args []string) (out string, err error) {
 		}
 	}
 
+	var fk, rk *sym.Key
+	if len(args) > 2 {
+		fkb, err := base58.Decode(args[2])
+		if err != nil {
+			return "", nil
+		}
+		fk, err = sym.NewKey(fkb)
+		if err != nil {
+			return "", nil
+		}
+	}
+	if len(args) > 3 {
+		rkb, err := base58.Decode(args[3])
+		if err != nil {
+			return "", nil
+		}
+		rk, err = sym.NewKey(rkb)
+		if err != nil {
+			return "", nil
+		}
+	}
```

Comment:
The intention isn't to fail but to ignore invalid keys? (`return "", nil` on `err != nil`)

Reply:
Typo! Fixing...